src/Compilers/Core/Portable/Compilation/ParseOptions.cs

```diff
@@ -32,9 +32,13 @@ public abstract class ParseOptions
         public SourceCodeKind SpecifiedKind { get; protected set; }
 
         /// <summary>
-        /// Gets a value indicating whether the documentation comments are parsed.
+        /// Gets a value indicating whether the documentation comments are parsed and can produce diagnostics, parsed only, or treated as regular comments.
```

Comment:
Consider simply saying: "Gets a value indicating whether the documentation comments are parsed and analyzed."